Gopkg.toml

```diff
@@ -24,6 +24,11 @@
 #   go-tests = true
 #   unused-packages = true
 
+required = [
+    "github.com/gogo/protobuf/protoc-gen-gogoslick",
```

Comment:
Fix version of `github.com/gogo/protobuf/protoc-gen-gogoslick` in `install-build-tools` section Makefile

Reply:
done